src/vue/common/IdleMessage.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="idle-message-wrapper">
+    <div class="idle-message">
+      <p class="idle-message__text">
+        {{ "idle-message.notification-message" | globalize }}
+      </p>
+      <div class="idle-message__btn-wrp">
+        <button class="idle-message__btn" @click="closeSelf">
+          {{ "idle-message.close-lbl" | globalize }}
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+const EVENTS = {
+  updateIsShown: 'update:isShown',
+}
+
+export default {
+  name: 'idle-message',
+  props: {
+    isLoggedout: { type: Boolean, default: false },
+  },
+  methods: {
+    closeSelf () {
+      this.$emit(EVENTS.updateIsShown, false)
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+@import "~@scss/variables";
+@import "~@scss/mixins";
+
+.idle-message-wrapper {
+  display: inline-block;
+  margin-left: 50px;
+  margin-right: -30px; //hack
+
+  @include respond-to($x-small) {
+    display: block;
+    margin: 0 auto 2rem;
+  }
+}
+
+.idle-message {
```

Comment:
positioned weirdly, looks like status message but actually isn’t, should be highlighted somehow, not white backgrounded.

I still pref something like .kyc-request-state with pendinng state of `src/vue/pages/Verification.vue`